src/main/resources/db/changelog/db.changelog-RDM-9052.xml

```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    
+    <changeSet id="RDM-9052-1" author="henry">
+	    <preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">SELECT count(c.relname) FROM pg_class c WHERE c.relkind = 'S' and c.relname = 'user_profile_audit_id_seq'</sqlCheck>
```

Comment:
is this checking whether the sequence  does not exist? could we simplify by using <sequenceExists>
as documented in: 
https://docs.liquibase.com/concepts/advanced/preconditions.html

Reply:
<sequenceExists> tag is not working for this library version.